starburst-presto/presto.sh

```diff
@@ -21,77 +21,105 @@ export PATH=/usr/bin:$PATH
 readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
 readonly PRESTO_MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
 readonly WORKER_COUNT=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count)
-readonly PRESTO_BASE_URL=https://storage.googleapis.com/starburstdata/presto
-readonly PRESTO_MAJOR_VERSION="312"
-readonly PRESTO_VERSION="${PRESTO_MAJOR_VERSION}-e.1"
-readonly HTTP_PORT="$(/usr/share/google/get_metadata_value attributes/presto-port || echo 8080)"
+readonly PRESTO_MAJOR_VERSION="$(/usr/share/google/get_metadata_value attributes/presto_major_version || echo '323')"
+readonly STARBURST_PRESTO_VERSION="$(/usr/share/google/get_metadata_value attributes/sbpresto_version || echo '323-e.3')" 
+readonly HTTP_PORT="$(/usr/share/google/get_metadata_value attributes/presto-port || echo 8060)"
 readonly INIT_SCRIPT="/usr/lib/systemd/system/presto.service"
-PRESTO_JVM_MB=0
-PRESTO_QUERY_NODE_MB=0
+PRESTO_JVM_MB=0;
+PRESTO_QUERY_NODE_MB=0;
 # Allocate some headroom for untracked memory usage (in the heap and to help GC).
-PRESTO_HEADROOM_NODE_MB=256
+PRESTO_HEADROOM_NODE_MB=256;
+
+# Add GCS Hive connector Jar
+if [[ -d /usr/local/share/google/dataproc/lib ]]; then
+  readonly CONNECTOR_JAR="$(find /usr/local/share/google/dataproc/lib -name 'gcs-connector-*.jar')"
```

Comment:
gcs-connector jar comes builtin in Presto Hive connector

https://github.com/prestosql/presto/blob/7aaa0f5fac1454b23e4bd2ac3a511d749bd00fbe/presto-hive/pom.xml#L173-L174

@medb am i correct we don't need to update it and we can just use `hadoop2-2.0.0` version?

Reply:
Yes, there no need to update it manually.